Taskfile.yml

```diff
@@ -732,7 +732,7 @@ tasks:
     cmds:
       - "kubectl create namespace cert-manager"
       - "kubectl label namespace cert-manager cert-manager.io/disable-validation=true"
-      - "kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.8.2/cert-manager.yaml"
+      - "kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.12.1/cert-manager.yaml"
```

Comment:
I think there are a bunch of other references to cert manager elsewhere in the repo, such as in our readme - can you fix those up too?

Reply:
Done!